src/test/java/io/strimzi/kafka/quotas/StaticQuotaCallbackTest.java

```diff
@@ -341,7 +343,9 @@ void staticQuotaMetrics() {
                 StaticQuotaConfig.PRODUCE_QUOTA_PROP, 15.0,
                 StaticQuotaConfig.FETCH_QUOTA_PROP, 16.0,
                 StaticQuotaConfig.REQUEST_QUOTA_PROP, 17.0,
-                StaticQuotaConfig.ADMIN_BOOTSTRAP_SERVER_PROP, "localhost:9092"
+                StaticQuotaConfig.ADMIN_BOOTSTRAP_SERVER_PROP, "localhost:9092",
+                // disable the plugin, so we don't need to configure it completely
+                StaticQuotaConfig.STORAGE_CHECK_INTERVAL_PROP, 0
```

Comment:
I think this change can be removed since #48 was merged

Reply:
Good catch, forgot to rebase it and remove those lines. Thanks